### ISSUE TITLE
"Popeye" deformity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10812]
**Preferred term label:**
"Popeye" deformity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Visible biceps muscle belly retraction proximally or distally secondary to biceps tendon tear, more noticeable with elbow flexion

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Distal biceps tendon tear
Proximal biceps tendon tear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001366
- label: Popeye sign
- parents: [Abnormal joint physiology (HP:0034430)](https://hpo.jax.org/app/browse/term/HP:0034430)
- pmid: PMID:35288410
- def: Exaggerated bulging of the belly of the biceps muscle upon contraction.
- comment: The Popeye sign involves hump formation caused by bulging of the biceps muscle because of biceps tendon tear. The bulging is more noticeable with elbow flexion and is said to resemble the biceps muscle of the cartoon character Popeye.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10812
- synonyms:
